src/TitleDetails/BoundPiecesList/BoundPiecesList.js

```diff
@@ -1,41 +1,33 @@
 import PropTypes from 'prop-types';
-import { useMemo } from 'react';
+import {
+  useMemo,
+  useState,
+} from 'react';
 
 import {
   acqRowFormatter,
   PrevNextPagination,
 } from '@folio/stripes-acq-components';
-import { MultiColumnList } from '@folio/stripes/components';
+import {
+  Loading,
+  MultiColumnList,
+} from '@folio/stripes/components';
 import { useStripes } from '@folio/stripes/core';
 
 import { PIECE_COLUMN_MAPPING } from '../constants';
-import { usePiecesList } from '../hooks';
 import {
   BOUND_ITEMS_LIMIT,
   VISIBLE_COLUMNS,
 } from './constants';
+import { useItemsList } from './hooks';
 import { getColumnFormatter } from './utils';
 
-export const BoundPiecesList = ({
-  filters,
-  id,
-  title,
-}) => {
+export const BoundPiecesList = ({ id, title }) => {
   const stripes = useStripes();
-  const {
-    isFetching,
-    pagination,
-    pieces,
-    setPagination,
-    totalRecords,
-  } = usePiecesList({
-    filters,
-    limit: BOUND_ITEMS_LIMIT,
-    title,
-    queryParams: {
-      isBound: true,
-    },
-  });
+  const [pagination, setPagination] = useState({ limit: BOUND_ITEMS_LIMIT, offset: 0 });
+  const { isFetching, items } = useItemsList({ titleId: title.id, poLineId: title.poLineId });
+
+  const totalCount = items.length;
```

Comment:
How could the total count be equal to `items.length` if it is only a chunk (paginated) of data

Reply:
`responses.flatMap` chunks and the length will be defined as total records https://github.com/folio-org/ui-receiving/pull/540/files#diff-61cd822aacf0fa144452c8111c3ae22154dab8a3221c12a4d2a24c0ebbaf134eR41